packages/home-service/src/types.d.ts

```diff
@@ -1,30 +1,39 @@
 declare module '*.graphql';
 declare module '*.json';
 
+declare enum Role {
+  admin = "ADMIN",
+  user = "USER"
```

Comment:
Keep it all caps, for maintaining consistency between the typedef.graphql and typescript types.
```suggestion
  ADMIN = "ADMIN",
  USER = "USER"
```